AssetRipper.Library/Exporters/Shaders/UnityPatch/YamlShaderPostprocessor.txt

```diff
@@ -0,0 +1,25 @@
+using UnityEngine;
+using UnityEditor;
+
+namespace AssetRipperPatches.Editor
+{
+    /// <summary>
+    /// This script is AssetRipper's patch for shaders exported as YAML assets.
+    /// Such a shader can be assigned to and used by a material as a regular .shader asset,
+    /// but it does not work with Shader.Find() unless we explicitly register it. 
```

Comment:
I saw you document on the other file that this is only applicable to 2018 +. I would also like it mentioned here.